.registry/resources/minimalgcp.ui-schema.yaml

```diff
@@ -47,7 +47,7 @@ configSections:
   items:
   - name: additionalInfoSnippet
     controlType: codeSnippet
-    codeText: curl -s https://raw.githubusercontent.com/crossplaneio/stack-gcp-minimal/master/hack/gcp-credentials.sh | PROJECT_ID=[PROJECT_ID] bash -
+    codeText: curl -s https://raw.githubusercontent.com/crossplane/stack-minimal-gcp/master/hack/gcp-credentials.sh | PROJECT_ID=[PROJECT_ID] bash -
```

Comment:
I think the earlier version of this URL was not correct as well. What do you think about directly linking to https://github.com/crossplane/stack-gcp/blob/master/examples/gcp-credentials.sh to avoid duplication?

Reply:
the script file referenced is a variation of the file in master which is why it was in a hack file. It is specific to the purposes of using it in this context